docs/rfcs/2024-12-06-chat-tags.md

```diff
@@ -0,0 +1,25 @@
+# Chat tags, emoji entry
+
+## Problem
+
+Organize chats.
+
+Stop users from using chat profiles to organize chats.
+
+More fun as we solve groups problem.
+
+## Solution
+
+This is a simple "quality of life" improvement that's long due, as the number of users and conversations grows, that would probably take just a week to ship.
+
+Proposed UX:
+- tags are per profile.
+- preset tags for favourites, contacts, groups and business chats.
+- user defined tags *must* have text and emoji.
+- if some conversation tag (preset or user defined) is not present in the conversation, the tag will not show in the bar, but would still allow selecting it when tagging.
```

Comment:
> the tag will not show in the bar

chat view bar?

Reply:
Don't quite get this one too, I think it means that if a tag has 0 matches in the conversation list it shouldn't show, please confirm @epoberezkin